Views/Config/form.html.php

```diff
@@ -79,35 +65,57 @@
         <?php echo $view['form']->row($form['isPublished']); ?>
         <hr />
         <?php echo $view['form']->row($form['apiKeys']); ?>
+        <?php
+        if ($useAuthorizationUrl):
```

Comment:
```suggestion
        <?php if ($useAuthorizationUrl): ?>
```
Let's use PHP tags in one line as it resembles HTML tags this way. The same for the endif on line 90.